.env.dev

```diff
@@ -3,14 +3,6 @@
 REACT_APP_FEATURE_FLAG_LENDING=disabled
 
 /* DEVELOPMENT */
-// Local
-
-REACT_APP_BITCOIN_NETWORK="regtest"
-REACT_APP_DEFAULT_ACCOUNT_SEED="//Alice"
-REACT_APP_PARACHAIN_URL="ws://127.0.0.1:9944"
-REACT_APP_FAUCET_URL="http://localhost:3035"
-REACT_APP_RELAY_CHAIN_NAME="kusama"
-DOCKER_RELAY_CHAIN_CURRENCY="KSM"
```

Comment:
Let's not remove the local environment variables, it is still useful to run local parachain when we need to develop against runtime version that is not deployed to testnet (for AMM for example).

Reply:
Makes sense—I'll add a comment (or change the header) as I think 'local' gives the impression it's what you would use to run the UI locally, whereas we're suggesting users run it against one of the testnets.